eureka_ml_insights/metrics/metrics_base.py

```diff
@@ -144,13 +144,39 @@ def __evaluate__(self, answer_text, target_text, is_valid):
         else:
             return "incorrect"
 
+class ExactOrMatch(ExactMatch):
+    """This class checks for a case-sensitive, but otherwise exact match, and returns the or of them."""
```

Comment:
This sentence is unclear to me. What does "case sensitive but otherwise exact match" mean? What are "them"?

Reply:
will clarify

Reply:
Removed this class